### PR TITLE
Tweaking the forgot password and reset password pages

### PR DESCRIPTION
**Pull request by tonyxj**
Guys, @nikilselvam @psybuzz 

It's been a long time coming. ~~~
Hope y'all are doing well!

This PR addresses the forget password page as well as the reset password page. Mostly just aesthetics tweaks to make the pages more consistent with our existing style.

### CODE REVIEW

File: views/resetPassword.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
 		</div>
 		<div class="centered">
 			<br><br>
-			<span>Need further assistance?  Contact <a href="mailto:goldenconversations@gmail.com">us by email</a><br>We'll do our best to help!</span>
+			<span>Need further assistance?  Contact us <a href="mailto:goldenconversations@gmail.com">by email</a>.<br>We'll do our best to help!</span>
```
Comment (psybuzz) [inline views/resetPassword.html]: Nice touch.

**Comment by psybuzz** [comment]
LGTM!

**Comment by tonyxj** [comment]
Great! Thanks @psybuzz !
What do you think of the PR @nikilselvam? :)

**Comment by nikilselvam** [comment]
Apologies for the late response on this one, guys, but LGTM!

**Pull request merged**